pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -230,12 +231,15 @@ func (c *Controller) reconcile(qualifiedName util.QualifiedName) util.Reconcilia
 	}
 
 	typeConfig.Status.ObservedGeneration = typeConfig.Generation
-	if syncRunning {
+	syncControllerRunning := startNewSyncController || (syncRunning && !stopSyncController)
```

Comment:
(No action required) Would it be worth only calling `UpdateStatus` if status had actually changed?  As written it looks like status will be set everytime an ftc is reconciled.  If so, I'm assuming it can wait for the refactor you mentioned in #801.